.github/scripts/fetch_prs.mjs

```diff
@@ -51,56 +56,126 @@ async function fetchReviewerDetails(login) {
+    }
+  }
+
+  // Add reviewers from reviews
+  const reviews = await fetchReviewsForPR(pr.number);
+  const uniqueReviewers = new Set(fetchedReviewers); // To avoid duplicates
+  
+  for (const review of reviews) {
+    const details = await fetchReviewerDetails(review.user.login);
+    const rewName = details.name ? details.name.replace(/ /g, '.').toLowerCase() : 'No name';
+    if (!uniqueReviewers.has(`${details.login} (@${rewName})`)) {
+      fetchedReviewers.push(details ? `${details.login} (@${rewName})` : review.user.login);
+      uniqueReviewers.add(`${details.login} (@${rewName})`);
+    }
+  }
+
+  if (fetchedReviewers.length > 0) {
+    reviewersInfo = `Reviewers: ${fetchedReviewers.join(', ')}`;
   }
 
   return `- pr${pr.number}: [${pr.title}](${pr.html_url}) (Created: ${moment(pr.created_at).fromNow()}) - ${reviewersInfo}`;
+  // | ${pr.labels.map(label => label.name).join(', ')}${additionalInfo}`;
 }
 
+  // const mergeStatus = isReadyForMerge ? '| :white_check_mark: **Ready for merge** :arrow_forward:' : '';
+  
+  // if (pr.requested_reviewers && pr.requested_reviewers.length > 0) {
+  //   const reviewers = await Promise.all(
+  //     pr.requested_reviewers.map(async reviewer => {
+  //       const details = await fetchReviewerDetails(reviewer.login);
+  //       const rewName = details.name ? details.name.replace(/ /g, '.').toLowerCase() : 'NoName'
+  //       return details ? `${details.login} (@${rewName})` : reviewer.login;
+  //     })
+  //   );
+  //   reviewersInfo = `Reviewers: ${reviewers.join(', ')}`;
+  // }
+
+  // return `- pr${pr.number}: [${pr.title}](${pr.html_url}) (Created: ${moment(pr.created_at).fromNow()}) - ${reviewersInfo} ${mergeStatus}`;
+// }
+
 async function generateSummary(prs) {
   const now = moment();
-  const reviewRequired = prs;
-  const recent = reviewRequired.filter(pr => moment().diff(moment(pr.created_at), 'days') <= recentDays);
-  const lasting = reviewRequired.filter(pr => moment().diff(moment(pr.created_at), 'days') > recentDays);
+  const recent = [];
+  const lasting = [];
+  const readyForMerge = [];
+
+  for (const pr of prs) {
+    const reviews = await fetchReviewsForPR(pr.number);
+    const approvals = reviews.filter(review => review.state.toLowerCase() === 'approved');
+    const isReadyForMerge = approvals.length >= approvalsRequired;
+
+    if (isReadyForMerge) {
+      readyForMerge.push(pr);
+    } else if (moment().diff(moment(pr.created_at), 'days') <= recentDays) {
+      recent.push(pr);
+    } else {
+      lasting.push(pr);
+    }
+  }
 
   let summary = `## ${project} PRs ${now.format('YYYY-MM-DD')}\n\n`;
 
-  if (reviewRequired.length === 0) {
+  if (prs.length === 0) {
     summary += `:tada: No review required for today\n`;
     return summary;
   }
 
+  if (readyForMerge.length > 0) {
+    const readyForMergePRsInfo = await Promise.all(readyForMerge.map(pr => formatPR(pr, ' - **Ready for merge**')));
```

Comment:
Second argument is not used (it is commented in formatPR), also, not needed for PRs in a separate section.

```suggestion
    const readyForMergePRsInfo = await Promise.all(readyForMerge.map(pr => formatPR(pr)));
```